build-monitor-acceptance/build-monitor-acceptance-base/src/main/java/net/serenitybdd/integration/jenkins/JenkinsInstance.java

```diff
@@ -65,8 +65,10 @@ public String pluginUnderTestVersion() {
         return pluginUnderTest.version();
     }
 
+    //todo investigate impact of this
```

Comment:
@jan-molak what do you think of this change? I had to do it to make the test run everything for a newer jenkins than the plugin was compiled with.